index.html

```diff
@@ -3857,6 +3857,73 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media type used by this specification relies on JSON-LD, JSON-LD
+processing is not required to be performed, since JSON-LD is JSON. Some
+scenarios where processing a <a>verifiable credential</a> or a <a>verifiable
+presentation</a> as JSON is desirable include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+Prior to verification when using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into systems that store or index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed for securing mechanisms that require an understanding of
+and processing of JSON-LD.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
+          </p>
+
+          <ul>
+            <li>
+Ensure that all values associated with a `@context` property are in the
+expected order, the contents of the context files match known good
+cryptographic hashes for each file, and the contents are known to be valid.
+            </li>
+          </ul>
+
+          <p>
+Using static context files with a JSON Schema is one acceptable approach to
+implementing the rule above. This can ensure proper term identification,
+typing, and order, when a JSON document is processed as JSON-LD.
+          </p>
+
+          <p>
+The rule above guarantees semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD processors will use the
+specific mechanism provided and can verify that all terms are correctly
+specified, JSON-based processors implicitly accept the same set of terms without
+testing that they are correct. In other words, the context in which the data
+exchange happens is explicitly stated for both JSON and JSON-LD by using the
+same mechanism. With respect to JSON-based processors, this is achieved in a
+lightweight manner, without having to use JSON-LD processing libraries.
+          </p>
+
```

Comment:
```suggestion
```

I elaborated in me previous comment above, but saying that `the terms "in the valid JSON-LD document" have one meaning whether they are interpreted as JSON-LD or if they're interpreted as JSON "when they follow certain rules"` is very misleading and I am against us saying so.

and if "pure JSON processing" of a conforming document is allowed (even if under the certain conditions), `@context` should rather be made optional.

Reply:
I would see that "pure json" and "required `@context`" can be split into two different topics.

The first, "pure json", can be seen as inherited property/feature of JSON-LD. If you would like to change this, please submit amendment into JSON-LD.

> JSON-LD is designed to be usable directly as JSON, with no knowledge of RDF [[RDF11-CONCEPTS](https://www.w3.org/TR/json-ld11/#bib-rdf11-concepts)].

> A JSON-LD document is always a valid JSON document. This ensures that all of the standard JSON libraries work seamlessly with JSON-LD documents.

The second, "required `@context`", can be seen as VCDM2.0 data structure enforcement on top of underlying data format. This is normal use and compliant with the data formats.

I agree that "pure" is a new description and "vanilla" has also be seen as replacement, but neither work as good as RFC reference.

Reply:
The "pure JSON" language has been removed and replaced with @OR13's suggestion to just speak about "JSON" and "JSON-LD".

Reply:
@Sakurann I  am not seeing support to remove the examples (see multiple thumbs ups on @mtaimela's comment) and multiple change suggestions improving the content (not removing it).